signals/content/subject_contains_org_display_name.yml

```
@@ -0,0 +1,4 @@
+name: "Subject: Matches entry in the $org_display_names list"
+type: "query"
+source: |  
+  subject.subject in~ $org_display_names
```

```suggestion
  any($org_display_names, strings.icontains(subject.subject, . ))
```
The existing logic would have to be an exact char match (case insens). I'm guessing by the name that's what you want, but I'd wager a contains would be more helpful?